### DISCUSSION TITLE
TP3 Duda ejercicio 2.5 - ¿que pasaria si una cadena es mas grande que la otra?

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#106] [Preguntas y respuestas] [answered]
titulo

**Comment by martinvilu** [comment, marked as answer]
Ah, que buena pregunta.

Algo para tener en cuenta, como cadena segura, debería de funcionar.

Es más, recuerda que el tamaño y la capacidad son dos cosas diferentes.
